client-app/ui-kit/components/atoms/breadcrumbs/vc-breadcrumbs.vue

```diff
@@ -23,12 +23,9 @@
 </template>
 
 <script setup lang="ts">
-import { PropType } from "vue";
+export interface Props {
```

Comment:
you can remove extra 'export ' word

Reply:
I can't. If you use `defineProps` with an interface as generic type argument, then `vue` require to make it exportable. Otherwise you will get runtime error

Reply:
Done